tekton/resources/nightly-tests/bastion-p/deploy_tekton_component.yaml

```diff
@@ -0,0 +1,48 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: deploy-tekton-project-nightly
+spec:
+  workspaces:
+  - name: k8s-config
+    description: workspace to get k8s config file
+    mountPath: /root/.kube
+  - name: registry-credentials
+    description: workspace to get registry credentials
+    mountPath: /tekton/home/.docker
+  - name: registry-certificate
+    description: workspace to get registry self-signed certificate
+    mountPath: /opt/ssl/certs
+  - name: source-code
+    description: workspace with source code for tekton component
+    mountPath: /workspace
+  params:
+  - name: package
+    description: package to install
+  - name: container-registry
+    description: container registry used to publish build images
+  - name: target-arch
+    description: target architecture for tests (s390x, ppc64le, arm64)
+  steps:
+  - name: deploy
+    workingdir: /workspace/src/$(params.package)
+    image: gcr.io/tekton-releases/dogfooding/test-runner:latest
+    env:
+    - name: GOPATH
+      value: /workspace
+    - name: KO_DOCKER_REPO
+      value: $(params.container-registry)
+    - name: KUBECONFIG
+      value: /root/.kube/config
+    - name: SSL_CERT_FILE
+      value: /opt/ssl/certs/registry.crt
+    command:
+    - /bin/bash
+    args:
+    - -ce
+    - |
+      # busybox binary in distroless image doesn't work on ppc64le
+      # https://bugs.busybox.net/show_bug.cgi?id=13491
+      sed -i 's/gcr.io\/distroless\/base@sha256:[a-zA-Z0-9]*/registry.access.redhat.com\/ubi8\/ubi-minimal:latest/g' config/controller.yaml
```

Comment:
This is for the shell image, right?

Reply:
Yes, it is. Even though `distroless/base` image is available for `ppc64le` , the `busybox` [binary present](https://github.com/GoogleContainerTools/distroless/pull/622/files#diff-5493ff8e9397811510e780de47c57abb70137f1afe85d1519130dc3679d60ce5R423) in the image is for big endian ppc machines and fails on ppc64le(which is little endian). We are working on having a permanent fix via using debian based busybox incorporated in distroless image.

Reply:
Posted on that distroless PR too - https://github.com/GoogleContainerTools/distroless/pull/622#issuecomment-795077067 checking if debian repo way to install ppc64le based `busybox` can be option in `distroless/base` image .

Reply:
That's great, thank you!
This means that the nightly / full releases we produce for ppc64le are somewhat broken, right? 
Do we have an issue to track this on tekton side? We could have a separate image / release file for `ppc64le`, or we could wait until we have a working `busybox`, but it would be good to document that it's not working for now?

Reply:
> This means that the nightly / full releases we produce for ppc64le are somewhat broken, right?

Yes, since we have to replace `distroless` with `ubi`, we can say its broken for `ppc64le`. 
> Do we have an issue to track this on tekton side?

I have created https://github.com/tektoncd/pipeline/issues/3820 to track this.